Tests broken (maybe openrpc changed?)

A test regarding openrpc fails (see https://github.com/pymeasure/pyleco/actions/runs/10473269206):
~~~
__________________ Test_discover_method.test_fail[RPCServer1] __________________

self = <test_rpc_server.Test_discover_method object at 0x7f83c6102040>
methods = [{'examples': [{'params': [{'name': 'arg', 'value': None}], 'result': {'value': {}}}], 'name': 'sem', 'params': [{'nam...ue, 'schema': {'title': 'Arg1', 'type': 'number'}}], 'result': {'name': 'result', 'schema': {'title': 'Result'}}, ...}]

    def test_fail(self, methods: list):
        m = methods[2]
        assert m["name"] == "fail"
>       assert m["description"] == fail.__doc__
E       KeyError: 'description'

tests/json_utils/test_rpc_server.py:167: KeyError
~~~

It only happens on Python >=3.9, suggesting, that it has to do with `openrpc` package (see #90 ).

--- a/pyleco/json_utils/rpc_server_definition.py
+++ b/pyleco/json_utils/rpc_server_definition.py
@@ -133,7 +133,12 @@ class RPCServer:
                 continue
             method_dict = {"name": name}
             if method.__doc__:
-                method_dict["description"] = method.__doc__
+                lines = method.__doc__.split("\n")
+                method_dict["summary"] = lines[0]
+                if lines[1:]:
+                    method_dict["description"] = " ".join(
+                        line.strip() for line in lines[1:] if line
+                    ).strip()
             methods.append(method_dict)
         result["methods"] = methods
         return result

--- a/tests/json_utils/test_rpc_server.py
+++ b/tests/json_utils/test_rpc_server.py
@@ -52,7 +52,11 @@ def side_effect_method(arg=None):
 
 
 def fail():
-    """Fails always."""
+    """Fail always.
+
+    This method fails always.
+    And has a description.
+    """
     raise NotImplementedError
 
 
@@ -164,7 +168,14 @@ class Test_discover_method:
     def test_fail(self, methods: list):
         m = methods[2]
         assert m["name"] == "fail"
-        assert m["description"] == fail.__doc__
+        assert m["summary"] == "Fail always."
+        assert m.get("description") == "This method fails always. And has a description."
+
+    def test_simple_description(self, methods: list):
+        m = methods[3]
+        assert m["name"] == "simple"
+        assert m["summary"] == "A method without parameters."
+        assert m.get("description") is None
 
     def test_rpc_discover_not_listed(self, methods: list):
         for m in methods:

